shell/browser/web_contents_preferences.cc

```diff
@@ -141,7 +141,7 @@ WebContentsPreferences::WebContentsPreferences(
   SetDefaultBoolIfUndefined(options::kDisableHtmlFullscreenWindowResize, false);
   SetDefaultBoolIfUndefined(options::kWebviewTag, false);
   SetDefaultBoolIfUndefined(options::kSandbox, false);
-  SetDefaultBoolIfUndefined(options::kNativeWindowOpen, false);
+  SetDefaultBoolIfUndefined(options::kNativeWindowOpen, true);
```

Comment:
you need to also change the default value in https://github.com/electron/electron/blob/968b30c9b4459e4f7030042b595f26926481e260/shell/browser/web_contents_preferences.cc#L481 due to the problem described in https://github.com/electron/electron/pull/18369

Reply:
please also cleanup https://github.com/electron/electron/blob/968b30c9b4459e4f7030042b595f26926481e260/shell/browser/web_contents_preferences.cc#L196-L202

Reply:
Can you expand on how `SetDefaults` should be cleaned up? We still need to enable `kNativeWindowOpen` when sandbox is true in case a caller explicitly sets it to false, this will override it.

Reply:
> you need to also change the default value in
> 
> https://github.com/electron/electron/blob/968b30c9b4459e4f7030042b595f26926481e260/shell/browser/web_contents_preferences.cc#L481
> 
> due to the problem described in #18369

also, wondering how we could write a test for this?

Reply:
> Can you expand on how `SetDefaults` should be cleaned up? We still need to enable `kNativeWindowOpen` when sandbox is true in case a caller explicitly sets it to false, this will override it.
You're right, ignore this comment. I actually added that :)